### PR TITLE
Florence 2018 instagram

### PR DESCRIPTION
**Pull request by jread-usgs**

### CODE REVIEW

**Comment by jread-usgs** [comment]
Part of #156

File: 6_visualize/src/prep_spark_line_fun.R [hunk 1]
```diff
@@ -98,7 +98,7 @@ prep_spark_funs_data <- function(stage_data, site_data, timestep_ind, spark_conf
         data_frame(dateTime = head(flood_stage_line$dateTime, 1), stage_normalized = flood_stage_va),
         flood_stage_line %>% select(dateTime, stage_normalized),
         data.frame(dateTime = tail(flood_stage_line$dateTime, 1), stage_normalized = flood_stage_va))
-
+
```
Comment (aappling-usgs) [inline 6_visualize/src/prep_spark_line_fun.R]: What do you have your whitespace handling set to?

File: vizstorm-GIF-Florence-2018.Rproj [hunk 2]
```diff
@@ -11,7 +11,3 @@ Encoding: UTF-8
 
 RnwWeave: knitr
 LaTeX: pdfLaTeX
-
-AutoAppendNewline: Yes
-StripTrailingWhitespace: Yes
```
Comment (aappling-usgs) [inline vizstorm-GIF-Florence-2018.Rproj]: oh, this is why the trailing whitespace is not being stripped

Comment (aappling-usgs) [reply vizstorm-GIF-Florence-2018.Rproj]: not a huge deal, but i do like how setting this to Yes reduces the number of whitespace changes we make in PRs

Comment (jread-usgs) [reply vizstorm-GIF-Florence-2018.Rproj]: oh, didn't realize this. this must be the default. I started a new project w/ an empty dir, and then pointed the remotes to github to pull and build from there. I probably should've kept the original .proj that got pulled down.

**Pull request merged**